dashi/src/lib/store.ts

```diff
@@ -1,4 +1,4 @@
-import { create } from "zustand";
+import { create } from "zustand/index";
```

Comment:
Looks weird.

Reply:
Just checked - this is not required.

```suggestion
import { create } from "zustand";
```